### PR TITLE
Unit testing regression metrics

### PR DESCRIPTION
**Pull request by vikash018**
to - @mail4umar @VerticaDataScientist 

cc - @oualib 

Please review it. I have put comments in testcases for mismatches and clarifications.

### CODE REVIEW

File: verticapy/tests_new/machine_learning/metrics/test_regression_metrics.py [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+        vpy_lr_pred_pdf = vpy_lr_pred_vdf.to_pandas()
+        vpy_lr_pred_pdf["citric_acid"] = vpy_lr_pred_pdf["citric_acid"].astype(float)
+        vpy_lr_pred_pdf["residual_sugar"] = vpy_lr_pred_pdf["residual_sugar"].astype(
+            float
+        )
+
+        X = vpy_lr_pred_pdf[["citric_acid", "residual_sugar", "alcohol"]]
+        y = vpy_lr_pred_pdf["quality"]
+
+        # ******************************** verticapy logic ***************************************
```
Comment (mail4umar) [inline verticapy/tests_new/machine_learning/metrics/test_regression_metrics.py:154]: You can create two separate functions for better readability.

**Review by mail4umar — APPROVED** [review]

**Comment by oualib** [comment]
@mail4umar can you please check carefully this PR? It seems @vikash018 is modifying some of your plotting files.

**Comment by oualib** [comment]
@vikash018 please always add the right label to the PR.

**Review by oualib — CHANGES_REQUESTED** [review]
@mail4umar please triple check this PR

File: verticapy/plotting/_plotly/machine_learning/logistic_reg.py
Comment (mail4umar) [inline verticapy/plotting/_plotly/machine_learning/logistic_reg.py:1]: I think you have an older version of this function. plz revert these changes for this file "logistic_reg.py"

File: verticapy/tests_new/conftest.py
Comment (mail4umar) [inline verticapy/tests_new/conftest.py:1]: plz check if you have applied black here. the lines seem long.

File: verticapy/tests_new/machine_learning/metrics/test_classification_metrics.py
Comment (oualib) [inline verticapy/tests_new/machine_learning/metrics/test_classification_metrics.py:1]: why Github had detected some changes on this file whereas I see none?

**Review by oualib — APPROVED** [review]

File: verticapy/tests_new/machine_learning/metrics/test_classification_metrics.py
Comment (vikash018) [reply verticapy/tests_new/machine_learning/metrics/test_classification_metrics.py:1]: I guess, Initially there were few tests added to this file. I removed it as we have separate branch for classification metrics - unit test - 578-unit-testing-classification-metrics.

Comment (oualib) [reply verticapy/tests_new/machine_learning/metrics/test_classification_metrics.py:1]: can you please find a way to remove this file from the PR so we can merge?

Comment (vikash018) [reply verticapy/tests_new/machine_learning/metrics/test_classification_metrics.py:1]: sure. I can delete this file (in local) and commit to this branch.

Comment (oualib) [reply verticapy/tests_new/machine_learning/metrics/test_classification_metrics.py:1]: where is the classification metrics file? I don't see it in the main repo

Comment (vikash018) [reply verticapy/tests_new/machine_learning/metrics/test_classification_metrics.py:1]: I could see - https://github.com/vertica/VerticaPy/tree/unit-testing-regression-metrics/verticapy/machine_learning/metrics

**Pull request merged**